{{cookiecutter.project_slug}}/Makefile

```diff
@@ -67,7 +67,7 @@ clean-code:  ### Delete code directory on Storage
 
 .PHONY: upload-data  ### Upload data directory to Storage
 upload-data:
-	neuro storage load -p -u -T $(DATA_PATH) $(DATA_PATH_STORAGE)
+	neuro cp -p -u -T $(DATA_PATH) $(DATA_PATH_STORAGE)
```

Comment:
`cp -r` I think

Reply:
And also, let's use full names of the options: https://github.com/neuromation/cookiecutter-neuro-project/issues/24

Reply:
Good catch, thanks!